main.py

```diff
@@ -121,8 +162,69 @@ def write_candidate(perm: Permuter, source: str) -> None:
             break
         except FileExistsError:
             pass
-    print(f"wrote to {fname}")
+    print(f"wrote to {fname}")   
+    
+def post_score(context: EvalContext, permuter: Permuter, cand: Candidate, exception: BaseException, profiler: Profiler) -> None:
+    if exception != None:
+        print(f"[{permuter.unique_name}] internal permuter failure.")
+        traceback.print_exc()
```

Comment:
have you tested that you can send exceptions across processes? otherwise, we could send traceback strings instead